Products/RhaptosPrint/RhaptosPrintTool.py

```diff
@@ -316,7 +329,7 @@ def getEpubDir(self):
     security.declarePublic(ManagePermission, 'getAlternateStyles')
     def getAlternateStyles(self):
         """Returns a list of different print styles. The default is the LaTex format.
-          These are, for now, hardcoded. Also, the id corresponds to a .xsl file in the getEpubDir()/xsl
+          These are, for now, hardcoded. Also, the id corresponds to a .css file in the getEpubDir()/css
```

Comment:
perhaps we should remove the "for now". 😉  j/k

Reply:
Nah, I have plans …